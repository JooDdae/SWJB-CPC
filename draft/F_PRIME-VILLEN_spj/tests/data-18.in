7168231489556966074001427373493049112920891275807892913677727846120997013436165472437422998111171187788612937589186152316132957766913992221826357750238017977561906760294871163994114712887100977174363573710710922377417490346513709491869950598573321745619479679641181336198597883423196973739779296883797578358512161322173331695189313799727072791694783991930371913279115796793
