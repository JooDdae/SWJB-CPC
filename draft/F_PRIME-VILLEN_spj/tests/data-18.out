186
4 0 877 237 3 104 854 551 587 4 70 437 0 0 10 316 0 1 4 762 24 10 0 2 19 4 4 1 4 1058 317 126 2 14 18 845 1 14 4 579 1210 0 0 4 0 0 151 1104 10 2 4 3 24 284 3 1 2 0 18 2 10 3 4 972 222 1 1 1 0 1036 3 4 4 674 1010 25 4 3 18 1127 786 93 0 38 1226 0 15 0 1 154 169 4 4 0 4 596 3 4 2 4 28 29 1 1 2 4 4 13 4 655 629 2 127 94 1084 677 148 2 2 1 0 4 618 161 1195 116 42 2 2 114 149 4 153 580 1197 2 4 2 25 22 10 886 22 4 3 4 23 769 1 0 18 2 1 1 0 4 2 2 449 3 0 24 2 0 2 22 25 394 1 22 0 891 146 157 435 8 0 2 1 22 0 0 3 4 1195 2 